cmd/server/go.mod

```diff
@@ -0,0 +1,27 @@
+module server
+
+go 1.22.0
```

Comment:
[CHANGE REQUEST] You need to have just one.

Reply:
do you mean 1 module in the application? and not like now several modules